src/services/service.messaging.ts

```diff
@@ -105,7 +105,7 @@ function isMessageNew(
   oldMessages: IMessageMapById,
   messageId: Message['messageId']
 ): boolean {
-  return !oldMessages[messageId]
+  return !!messageId && !oldMessages[messageId]
```

Comment:
Why is this done?

Reply:
that's just a lint fix - messageId was potentially undefined